queries.sql

```diff
@@ -1,17 +1,99 @@
-/*Queries that provide answers to the questions from all projects.*/
+/*Queries that provide answers to the questions from  projects.*/
 
 SELECT * FROM animals WHERE name LIKE '%mon';
 
-SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31'
 
-SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT * FROM animals WHERE neutered =  true and escape_attempts < 3;
 
 SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 
-SELECT * FROM animals WHERE neutered =  true;S
+SELECT * FROM animals WHERE neutered =  true;
 
 SELECT * FROM animals WHERE name <> 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg BETWEEN  10.4 and 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN  10.4 and 17.3;
+
+-- transaction start
+
+ BEGIN;
+ 
+ UPDATE animals
+ SET species = 'unspecified';
+ 
+ ROLLBACK;
+ 
+ COMMIT;
+
```

Comment:
- [ ] Good Job adding the transaction as required, however, we need to verify that the change was made before and after the ROLLBACK. Kindly do so.

e.g
```
   BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```


### Note the select statements being used in the example to verify that the change was made. Do like was as per the requirement